docs/BR.md

```diff
@@ -853,30 +853,26 @@ The CA SHALL update and reissue CRLs at least (i) once every twelve months and (
 No stipulation.
 
 ### 4.9.9 On-line revocation/status checking availability
-OCSP responses MUST conform to RFC6960 and/or RFC5019. OCSP responses MUST either:
-
-1. Be signed by the CA that issued the Certificates whose revocation status is being checked, or
-2. Be signed by an OCSP Responder whose Certificate is signed by the CA that issued the Certificate whose
-revocation status is being checked.
-
-In the latter case, the OCSP signing Certificate MUST contain an extension of type id-pkix-ocsp-nocheck, as
-defined by RFC6960.
+The CA SHALL provide an OCSP responder that conforms to the profile specified by RFC5019 for all Certificates issued, including the transport profile.
 
 ### 4.9.10 On-line revocation checking requirements
-Effective 1 January 2013, the CA SHALL support an OCSP capability using the GET method for Certificates issued
-in accordance with these Requirements.
+All OCSP responses MUST conform to the profile specified in Section 7.3
 
-For the status of Subscriber Certificates:
+For newly issued Subscriber Certificates, OCSP responses SHOULD be published prior to the delivery of the Certificate to the Subscriber. OCSP responses MUST be published no later than 10 minutes after the Subscriber Certificate has been issued.
 
-The CA SHALL update information provided via an Online Certificate Status Protocol at least every four days. OCSP responses from this service MUST have a maximum expiration time of ten days.
+The CA SHALL publish an updated OCSP response for Subscriber Certificates by the earlier of:
+1. 8 hours prior to the nextUpdate field of the previously published OCSP response
+2. If the OCSP response has a validity period, as defined by the difference in time between the thisUpdate and nextUpdate fields, of greater than 16 hours, then the value of the thisUpdate field plus one half of the validity period of the OCSP response.
```

Comment:
This simply aligns with the most restrictive Browser Member's program requirements, which are documented at https://social.technet.microsoft.com/wiki/contents/articles/31633.microsoft-trusted-root-program-requirements.aspx#C_Revocation_Requirements

So while this is a change in the BRs, it is not a change for any Microsoft-trusted CAs, so this hopefully should be reasonable to both CAs and other Browser members.

I'm slightly concerned about reusing the term "validity period" in the context of the OCSP response, due to the Proper Term Validity Period related to Certificates, but I don't have any good suggestions here.